content/en/flux/releases/_index.md

```diff
@@ -0,0 +1,151 @@
+
+Patch releases do not contain breaking changes, feature additions or any type of user-facing changes.
+If a CVE fix requires a breaking change, then a minor release will provide the fix.
+
+We expect users to be running the latest patch release of a given minor release.
+
+### Minor releases
+
+Minor releases are intended for backward-compatible feature additions and improvements.
+Note that breaking changes may occur if required by a security vulnerability fix.
+
+Minor releases are used when updating the Flux controllers or Kubernetes dependencies
+from one minor version to another.
+
+In effect, this means a Flux minor version will be released at least every four months, after each
+Kubernetes minor version release. For in-depth information about this, please refer to the
+[release cadence](#release-cadence) section of this document.
+
+### Major releases
+
+Major releases are intended for drastic changes to the Flux behaviour or security stance.
+
+A Flux major release will be announced ahead of time throughout all communication channels,
+and a support window of one year will be provided for the previous major version.
+
+## Release cadence
+
+Flux is _at least_ released at the same rate as Kubernetes, following their cadence of three
+minor releases per year. After each Kubernetes minor release, the CLI and all controllers are
+tested against the latest Kubernetes version and are released approximately two weeks after Kubernetes.
+The newly released Flux version offers support for Kubernetes N-2 minor versions.
+
+It is worth noting that in certain scenarios where project dependencies are not in sync with
+the Kubernetes version or conflicts arise, this two-week timeframe may prove insufficient,
+requiring additional time to address the issues appropriately.
+
+Flux may have more than three minor releases per year if maintainers decide to ship a 
+new feature or optimization ahead of schedule.
+
+## Supported releases
+
+For Flux the CLI and its controllers, we support the last three minor releases.
+Critical bug fixes, such as security fixes, may be back-ported to those three minor
+versions as patch releases, depending on severity and feasibility.
+
+Note that back-porting is provided by the community on a best-effort basis.
+
+The Flux controllers are guaranteed to be compatible with each other
+within one minor version (older or newer) of Flux.
+
+The `flux` command-line tool is supported within one minor version (older or newer) of Flux.
+
+## Supported upgrades
+
+Users can upgrade from any `v2.x` release to any other `v2.x` release (the latest patch version).
+
+After upgrade, [Flux Custom Resources](controllers.md#api-versioning) may require editing,
+for which migration instructions are provided as part of the
+[changelog](#release-changelog).
+
+We expect users to keep Flux up-to-date on their clusters using automation tools
+such as [Flux GitHub Actions](../../action) and
```

Comment:
This link leads to https://deploy-preview-1543--fluxcd.netlify.app/action which doesn't exist.

Reply:
Nice catch! Fixed ty